pkg/apis/events/v1/zz_generated.conversion.go

```diff
@@ -58,12 +58,12 @@ func RegisterConversions(s *runtime.Scheme) error {
 		return err
 	}
 	if err := s.AddConversionFunc((*core.Event)(nil), (*v1.Event)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return Convert_core_Event_To_v1_Event(a.(*core.Event), b.(*v1.Event), scope)
```

Comment:
Thank you for your contribution, but these codes are automatically generated and we should not modify them.

Reply:
I revert these codes in the generated go file. Would this be good?